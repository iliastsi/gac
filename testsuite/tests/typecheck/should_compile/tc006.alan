unusedResult() : proc
  foo1() : int
  { return 42; }
{
  foo1();
}
