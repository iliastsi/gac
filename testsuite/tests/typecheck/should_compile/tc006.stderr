tc006.alan:5:3-9: Warning: Unused return value of function `foo1'

